tests/data/state_estimation/meshed-network-observability/01-observable-with-branch-measurement/params.json

```diff
@@ -0,0 +1,12 @@
+{
+  "calculation_method": ["newton_raphson", "iterative_linear"],
+  "rtol": 1e-8,
+  "atol": {
+    "default": 1e-8,
+    ".+_residual": 5e-4
+  },
+  "fail": {
+    "raises": "NotImplementedError",
+    "reason": "Observability check for meshed network is not implemented yet."
+  }
```

Comment:
do we want to merge this before merging #1049 ? if not, please update to the current format.